src/lib/createGHA/baseFile.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * The base GitHub Actions workflow file, which is used for generating the YAML file.
+ *
+ * The reason this is expressed as a JavaScript string as opposed to a YAML file is because
```

Comment:
Not totally accurate. You could add `path/to/file.yaml` into the `include` array in `tsconfig.json` and it'll get moved into `dist/` when you run `npm run build`. At least I think so!

Reply:
Yeah I had tried this and the `include` array only supports compiled (i.e. JS/TS/etc.) files 😕 see this issue I linked to: https://github.com/microsoft/TypeScript/issues/30835